-- Downgrade the minimum setuptools version to match the one available in the
-- FreeBSD ports tree and convert the license field to the PEP 621 table form.
--- pyproject.toml.orig	2026-10-02 06:29:38 UTC
+++ pyproject.toml
@@ -1,10 +1,10 @@
 [build-system]
-requires = ["setuptools>=64", "setuptools_scm>=8"]
+requires = ["setuptools>=63.1.0", "setuptools_scm>=8"]
 build-backend = "setuptools.build_meta"
 
 [project]
 name = "universal_pathlib"
-license = "MIT"
+license = {text = "MIT"}
 authors = [
     {name = "Andrew Fulton", email = "andrewfulton9@gmail.com"},
 ]
